### PR TITLE
Random fixes

### PR DESCRIPTION
**Pull request by xirreal**
Fixed hidden channels basic function.
Fixed avatar links and direct download not creating an entry in the context menu.
Fixed button in settings not opening the plugins folder.

### CODE REVIEW

**Comment by xirreal** [comment]
More fixes

**Comment by Xazii** [comment]
I can't install ED, help me pls: discord id: @! Xazi#6666 
Desktop Screenshot 2020 12 08 - 12 37 01 64 (2)

File: plugins/avatar_links.js [hunk 1]
```diff
@@ -27,7 +27,8 @@ module.exports = new Plugin({
         // Make sure it's already in the DOM
         await new Promise(r => {setTimeout(r, 5)});
         const theMenu = document.querySelector('.'+cm.menu);
-        const reactData = theMenu.__reactInternalInstance$;
+
+        const reactData = theMenu[Object.keys(theMenu)[0]];
```
Comment (rauenzi) [inline plugins/avatar_links.js:31]: It is not guaranteed to be the first key, this is not a reliable solution. This is more reliable:

```
theMenu[Object.keys(theMenu).find(key => key.startsWith("__reactInternalInstance") || key.startsWith("__reactFiber"))]
```

**Review by rauenzi — CHANGES_REQUESTED** [review]

**Review by rauenzi — APPROVED** [review]
LGTM

**Pull request merged**